cookbooks/fb_consul/README.md

```diff
@@ -0,0 +1,118 @@
+fb_consul Cookbook
+==================
+Manages a consul instance
+
+Requirements
+------------
+
+Attributes
+----------
+* node['fb_consul']['enable']
```

Comment:
The actual attribute you use is `enabled`, but most cookbooks use `enable`, so maybe rename it to that?

Reply:
You sure?
```
[phil@rider (chef13) cookbooks]$ grep enable */attributes/default.rb | grep "'enabled' => " | wc -l
5
[phil@rider (chef13) cookbooks]$ grep enable */attributes/default.rb | grep "'enable' => " | wc -l
18
```

Reply:
Yeah I don't really care that strongly :) but at least the README and the actual attribute should agree